pages/common/git-clone.md

```diff
@@ -30,3 +30,7 @@
 - Clone an existing repository only fetching a specific branch:
 
 `git clone --branch {{name}} --single-branch {{remote_repository_location}}`
+
+- Clone an existing repository using a specific SSH command:
+
+`git clone --config core.sshCommand="{{ssh -i path/to/private_ssh_key}}" {{remote_repository_location}`
```

Comment:
```suggestion
`git clone --config core.sshCommand="{{ssh -i path/to/private_ssh_key}}" {{remote_repository_location}}`
```

Reply:
Does the bot not spot these?

Reply:
No, it doesn't. https://github.com/tldr-pages/tldr-lint/issues/76